include/SFML/Graphics/Font.hpp

```diff
@@ -49,6 +55,7 @@ class ResourceStream;
 namespace sf
 {
 class InputStream;
+class Texture;
```

Comment:
This doesn't make sense when we're already including the full texture header above.

Reply:
Remnant of trying to remove the header, will remove.